pkgs/games/mindustry/default.nix

```diff
@@ -88,7 +88,7 @@ let
   '';
 
   # The default one still uses jdk8 (#89731)
-  gradle_6 = (gradleGen.override (old: { java = jdk; })).gradle_6_7;
+  gradle_6 = (gradleGen.override (old: { java = jdk; })).gradle_6_8;
```

Comment:
we can just use gradle_latest here (with the jdk override) now that 6 is the default

Reply:
We should verify the fixed-output derivations can still be built using the new version.

Reply:
If I can be of help here, please just tell me, otherwise I'll relax until you know exactly what you want to do :)

Reply:
Sure, @fgaz and I will discuss it and tell you when we have a good solution.

I think we should keep it set to a fixed version, so that we are notified when the version changes, because it's not so unlikely that gradle updates break the fixed-output derivation build without us noticing.
@fgaz do you agree or would you prefer to set it to latest regardless and just look out for gradle updates carefully?

Reply:
@petabyteboy I never saw a hash change happen on other gradle packages, but I agree that it could be more likely here. And we get notified anyway...

Reply:
Hmm, maybe I should also switch my other gradle packages to a fixed gradle